src/components/snackBar/SnackBar.tsx

```diff
@@ -78,15 +81,21 @@ export const SnackBar = ({
             <Snackbar
                 duration={duration}
                 visible={visible}
-                onDismiss={onDismiss}
+                onDismiss={onClose}
                 style={styles.snackBar}
             >
                 <View style={messageContainer as ViewStyle}>
                     <Text style={styles.message}>{message}</Text>
-                    <Pressable hitSlop={8} style={actionContainer as ViewStyle} onPress={onDismiss}>
-                        {actionLabel && <Text style={styles.actionLabel}>{actionLabel}</Text>}
-                        {iconName && <Icon name={iconName} color={theme.sw.colors.primary[50]} />}
-                    </Pressable>
+                    <View style={actionContainer as ViewStyle}>
+                        <Pressable hitSlop={8} onPress={onDismiss}>
```

Comment:
is 'OnDismiss' a good name for this ? What is the purpose of this pressable ?

Reply:
I like the way it is done in native paper with the action name and the Dismiss instead of the close:
```
      <Snackbar
        visible={visible}
        onDismiss={onDismissSnackBar}
        action={{
          label: 'Undo',
          onPress: () => {
            // Do something
          },
        }}>
        Hey there! I'm a Snackbar.
      </Snackbar>
```
Could we do something like that ?

Reply:
updated!